.github/workflows/d4l-ci-pull-request-validation.yml

```diff
@@ -10,11 +10,17 @@ on:
 jobs:
   pull-request-validation:
 
-    runs-on: [self-hosted, macos]
+    runs-on: ubuntu-latest
```

Comment:
switch to macos for everything that isnt user docker

Reply:
I mean it not necessary to switch, since as @wmontwe stated the linux variant suffices except something breaks.

Reply:
i think it makes it more robust in case we want to do ios frameworks and stuff since thats only possible on macOS

Reply:
I still think it is premature...but is then your call.

Reply:
done

Reply:
with `macos-latest` I can live.